tests/unit/test_table.py

```diff
@@ -1789,6 +1823,46 @@ def test_ctor_full_resource(self):
         assert instance.snapshot_time == expected_time
 
 
+class TestCloneDefinition:
+    @staticmethod
+    def _get_target_class():
+        from google.cloud.bigquery.table import CloneDefinition
+
+        return CloneDefinition
+
+    @classmethod
+    def _make_one(cls, *args, **kwargs):
+        klass = cls._get_target_class()
+        return klass(*args, **kwargs)
+
+    def test_ctor_empty_resource(self):
+        instance = self._make_one(resource={})
+        assert instance.base_table_reference is None
+        assert instance.clone_time is None
+
+    def test_ctor_full_resource(self):
```

Comment:
are `klass` and `ctor` known BQ terms? If not and if they're abbreviations or acronyms, please lengthen them to be a more human readable. If known BQ or similar terms, plz disregard

Reply:
These terms are used throughout the tests in this library, but I agree we could make them more human readable. Perhaps I can open a bug for doing that across all instances.